include/etiss/CPUCore.h

```diff
@@ -344,7 +344,8 @@ class CPUCore : public VirtualStructSupport, public etiss::ToString
      * @brief returns the list of all plugins.
      */
     inline std::list<std::shared_ptr<Plugin>> const *getPlugins() { return &plugins; };
-
+  float startTime;
```

Comment:
these no longer need to be members right? they can just be local variables in cpucore.cpp